backend/siarnaq/api/compete/models.py

```diff
@@ -205,14 +205,20 @@ class Match(SaturnInvocation):
     """The maps to be played in this match."""
 
     alternate_order = models.BooleanField()
-    """Whether players should alternate orderGbetween successive games of this match."""
+    """Whether players should alternate order between successive games of this match."""
 
     is_ranked = models.BooleanField()
     """Whether this match counts for ranked ratings."""
 
     replay = models.UUIDField(default=uuid.uuid4)
     """The replay file of this match."""
 
+    challonge_id = models.IntegerField(blank=True, null=True)
```

Comment:
Feels somewhat annoying that this needs to be stored here. I wonder if we could instead use a `through` model on the `TournamentRound` foreign key to avoid clogging this model with tournament information.

Reply:
we looked into it and turns out you can't for manytomany models